sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyRequestParameters.cs

```diff
@@ -77,13 +77,19 @@ internal KeyRequestParameters(KeyType type, KeyCreateOptions options = default)
         internal KeyRequestParameters(EcKeyCreateOptions ecKey)
             : this(ecKey.KeyType, ecKey)
         {
-            Curve = ecKey.Curve;
+            if(ecKey.Curve != default)
+            {
+                Curve = ecKey.Curve;
+            }
```

Comment:
As I suggested in EcKeyCreateOptions perhaps we should make the Curve property nullable so we don't have to do this check. Also it seems this check might be somewhat invalid if the default value for KeyCurveName is valid it will result in Curve remaining "null"

Reply:
It actually is already nullable, I didn't check for that first. So I'll change it.
Thank you!!